Manifests/ManifestsApple/com.apple.mcxprinting.plist

```diff
@@ -0,0 +1,439 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!DOCTYPE plist PUBLIC "-//Apple//DTD PLIST 1.0//EN" "http://www.apple.com/DTDs/PropertyList-1.0.dtd">
+<plist version="1.0">
+	<dict>
+		<key>pfm_description</key>
+		<string>Printing settings</string>
+		<key>pfm_description_reference</key>
+		<string>BLANK</string>
```

Comment:
Should this stay _BLANK_ or should just be removed?

Reply:
I added content, thanks!